Since debian packages will be building for debian, and mach_o is only
relevant on OS X and iOS, We can remove the mach_o feature from
memsec.

If anyone wants to try to cross-build for OS X on debian, they should
figure out how to fix rust-mach-o-sys so that it does not fail on
architectures with an unsigned char (see
https://bugs.debian.org/953285)

--- a/Cargo.toml	2020-03-06 23:57:27.155352675 -0500
+++ b/Cargo.toml	2020-03-07 00:39:50.119049249 -0500
@@ -29,10 +29,7 @@
 alloc = ["getrandom"]
 default = ["use_os", "alloc"]
 nightly = []
-use_os = ["libc", "winapi", "mach_o_sys"]
-[target."cfg(any(target_os = \"macos\", target_os = \"ios\"))".dependencies.mach_o_sys]
-version = "0.1"
-optional = true
+use_os = ["libc", "winapi"]
 [target."cfg(unix)".dependencies.libc]
 version = "0.2"
 optional = true
